### DISCUSSION TITLE
Compatibility with whitecube Nova Flexible Content

### DISCUSSION THREAD

**Discussion by piotrh052b** [oneduo/nova-file-manager#113] [Q&A] [answered]
Hello

Is it possible to use Nova File Manage field inside whitecube Nova Flexible Content layout ?

**Comment by Rezrazi** [comment, marked as answer]
Hi

The package is compatible with whitecube's Nova Flexible Content package, you can use it like you'd use a regular native field.
If you do encounter a problem, please feel free to open an issue and we will gladly take a look at it.

All the best,
Charaf
